load_testing/lib/flow_ial2_proofing.py

```diff
@@ -44,110 +49,197 @@ def do_ial2_proofing(context):
+
+    selector = 'input[id="doc_auth_document_capture_session_uuid"]'
+    dcs_uuid = dom.find(selector).eq(0).attr("value")
 
-    files = {"doc_auth[front_image]": context.license_front,
-             "doc_auth[back_image]": context.license_back}
+    second_auth_token = authenticity_token(resp)
 
+    files = {"front": context.license_front,
+             "back": context.license_back,
+             }
+
+    if os.getenv("DEBUG"):
+        print("DEBUG: /api/verify/images")
     # Post the license images
+    resp = do_request(
+        context,
+        "post",
+        "/api/verify/images",
+        None,
+        None,
+        {
+            "flow_path": "standard",
+            "document_capture_session_uuid": dcs_uuid},
+        files,
+        None,
+        {"X-CSRF-Token": auth_token},
+    )
+
+    if os.getenv("DEBUG"):
+        print("DEBUG: /verify/document_capture")
     resp = do_request(
         context,
         "put",
         "/verify/document_capture",
-        "/verify/doc_auth/ssn",
-        "",
-        {"authenticity_token": auth_token, },
-        files
+        "/verify/ssn",
+        None,
+        {
+            "_method": "patch",
+            "doc_auth[document_capture_session_uuid]": dcs_uuid,
+            "authenticity_token": second_auth_token,
+        },
     )
     auth_token = authenticity_token(resp)
 
-    ssn = '900-12-3456'
-    # print("*** Using ssn: " + ssn)
+    ssn = f'900-12-{random.randint(0,9999):04}'
+    if os.getenv("DEBUG"):
+        print("DEBUG: /verify/ssn")
     resp = do_request(
         context,
         "put",
-        "/verify/doc_auth/ssn",
-        "/verify/doc_auth/verify",
-        "",
+        "/verify/ssn",
+        "/verify/verify_info",
+        '',
         {"authenticity_token": auth_token, "doc_auth[ssn]": ssn, },
     )
     # There are three auth tokens on the response, get the second
-    auth_token = authenticity_token(resp, 1)
+    auth_token = authenticity_token(resp, 0)
+
 
+
+    if os.getenv("DEBUG"):
+        print("DEBUG: /verify/verify_info")
     # Verify
     resp = do_request(
         context,
         "put",
-        "/verify/doc_auth/verify",
-        "/verify/phone",
-        "",
+        "/verify/verify_info",
+        None,
+        '',
         {"authenticity_token": auth_token, },
     )
-    auth_token = authenticity_token(resp)
 
+    # Wait until
+    for i in range(12):
+        if urlparse(resp.url).path == '/verify/phone':
+            # success
+            break
+        if urlparse(resp.url).path == '/backup_code_reminder':
+            # verify backup codes
+            if os.getenv("DEBUG"):
+               print("DEBUG: /backup_code_reminder")
+            auth_token = authenticity_token(resp)
+            resp = do_request(
+                context,
+                "get",
+                "/account?",
+                None,
+                '',
+                {"authenticity_token": auth_token, },
```

Comment:
probably a good candidate for a separate PR but this inconsistent spacing is making me think this would be a great project to introduce some linting into, we use [black](https://github.com/psf/black) in the identity-devops repo now and I've been a big fan of it
```suggestion
                { "authenticity_token": auth_token, },
```

Reply:
I love that idea. It was bothering me as well.

Reply:
It was simple enough, so I added it to this PR https://github.com/18F/identity-loadtest/pull/86/commits/47182e977a8f1e5151659e3dc698393596e0b47a

Reply:
Thanks! We should also add scripts and CI to ensure that code stays formatted!